public/content/developers/docs/storage/index.md

```diff
@@ -88,7 +88,7 @@ There aren't great tools to measure the level of decentralization of platforms,
 
 Decentralized tools without KYC:
 
-- 0Chain (implementing a non-KYC edition)
+- Züs (implementing a non-KYC edition)
```

Comment:
If a non-KYC option is not available, we should simply remove this. Can update when one is available.